Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/App.java

```diff
@@ -19,10 +20,15 @@ public void onCreate() {
                 .repositoryModule(new RepositoryModule())
                 .stringProviderModule(new StringProviderModule(this.getApplicationContext()))
                 .validationModule(new ValidationModule())
+                .filePickerModule(new FilePickerModule())
                 .build();
     }
 
     public AppComponent getAppComponent() {
         return appComponent;
     }
+
+    public void setComponent(AppComponent appComponent) {
```

Comment:
Really nice :) There is an annotation on setComponent method in DaggerMock lib example (I just forget to add it) but this solution is much better.